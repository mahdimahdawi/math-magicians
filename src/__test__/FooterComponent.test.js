import { render, screen } from '@testing-library/react';
import Footer from '../components/footer';

describe('Testing Footer Component', () => {
  test('testing footer', () => {
    render(<Footer />);

    // eslint-disable-next-line testing-library/await-async-query
    expect(screen.findAllByTitle('All Right Reserved &copy;')).toMatchSnapshot();
  });
});