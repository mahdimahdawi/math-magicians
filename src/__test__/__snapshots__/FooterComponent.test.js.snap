// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer Component testing footer 1`] = `Promise {}`;
